upgrade_versions.py

```diff
@@ -214,8 +234,17 @@ def _read_version_from_repo(self, repo_path):
 
                 with open(version_file, 'r') as f:
                     match = re.search(regex, f.read())
+                    match_groups = list(match.groups())
+
+                last_group = match_groups[-1]
+                for term in PRERELEASE_TERMS:
+                    if term in last_group:
+                        self._prerelease = last_group.replace('-','.')
```

Comment:
The prerelease in kernel's Makefile is '-rc4'. We can't have the '-' in the Release field of the spec file.